### PR TITLE
Df 1464 event meta data

### PR DESCRIPTION
**Pull request by sebworks**
event-meta

Fixes Df1464 - Create event meta data (venue location and event date)

### CODE REVIEW

**Comment by anselmbradford** [comment]
@sebworks See inline comments. Looks good otherwise! You might check https://github.com/necolas/idiomatic-css (per https://github.com/cfpb/cfgov-refresh/pull/254#issuecomment-74887567) for the CSS and tidy up the spacing around brackets, key/values.

Also, I don't know anything about vCard so can't comment on that. Is there a precedent for using that on the site and if not is it clearly the standard for address markup?

File: static/css/event-meta.less [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    letter-spacing:normal;
+  }
+}
+
+.event-meta_address{
+  display:inline-block;
+}
+
+.event-meta_date:before{
+  content : "\2014";
```
Comment (jimmynotjim) [inline static/css/event-meta.less]: What is the purpose of this pseudo elem? Isn't the year included in the `.event-meta_date` elem?

File: static/css/event-meta.less [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              span.event-meta_venue.p-extended-address
+            time
+              div.event-meta_date
+              div.event-meta_time
+  tags:
+    - cfgov-event-meta
+*/
+
+.event-meta {
+  .h5();
```
Comment (jimmynotjim) [inline static/css/event-meta.less]: This mixin adds a `text-transform: uppercase` but the mockup shows title case for the city and venue.

**Comment by jimmynotjim** [comment]
@Scotchester does the Bureau have any rules on using microformats.org vs schema.org? I know there was a discussion somewhere about improving SEO and either of these will help with that.

@sebworks Is this a fork of the calendar icon branch? We should only fork from the sprint branch to avoid issues. Right now we can't merge this without first merging the icon branch (and if that branch is updated this one will need to pull those commits in to avoid merge conflicts).

**Comment by Scotchester** [comment]
No rules on that of which I am aware. Not even sure if we're using either anywhere in existing projects. David Kennedy was advocating usage of them before he left, but I don't remember if he had a pref between microformats and schema.

**Comment by jimmynotjim** [comment]
Figured this a bigger discussion than this PR so I opened an [issue in the front-end repo](https://github.com/cfpb/front-end/issues/11)

File: static/css/event-meta.less (hunk 2 above)
Comment (sebworks) [inline static/css/event-meta.less]: @jimmynotjim, good catch. I missed that.

**Comment by sebworks** [comment]
@anselmbradford, so the dt-start class is only being used for the h-event microformat but I'll add a note for that.

**Comment by jimmynotjim** [comment]
@sebworks One last thing, Less indentation should be 4 spaces to match the existing files. Once that's done this should be good to merge.

**Comment by anselmbradford** [comment]
LGTM :+1: Not sure if there was resolution on vCard (pending https://github.com/cfpb/front-end/issues/11), but we can re-visit next week when `sprint-21` goes into `gh-pages`. cc @jimmynotjim

**Comment by jimmynotjim** [comment]
Should we (refresh team) come up with a rule for squashing tweak commits? I typically only squashed typos/debugging code in the past but I can see the benefit for squashing any adjustments for small PRs like these tasks.

**Comment by anselmbradford** [comment]
I think a guideline for our process isn't a bad idea, just to establish expectations of who's checking what when.
It could live in https://github.cfpb.gov/flapjack/Getting-started-with-Flapjack. Unless either of you are keen, I can throw something together this afternoon if I feel like I'm making moderate progress on my jira task.

Personally I like squashing all my commits down to one commit for a feature branch, and tie that commit to an issue. If by chance I have something outside of the issue committed in the same PR, I keep that as its own commit (e.g. https://github.com/cfpb/cfgov-refresh/pull/254). But I also do the squashing at the end of reviews as the step before merging, so it's important to know when that step has arrived. I also appreciate that getting work done trumps a clean git history and I hesitate to mandate a squashing step on anyone else's PR. So anyway, bottom line is I think a short guideline would be great, just so we don't discover each other's workflow expectations in a PR and instead can refer to a guideline.

**Pull request merged**